src/main/java/com/futurecollars/Task3/Task4/Age.java

```diff
@@ -0,0 +1,15 @@
+package com.futurecollars.Task3.Task4;
+
+public class Age {
+
+    public static boolean number(int Arg) {
+        if(Arg >= 18) {
+            return true;
+        }
+        else {
+            return false;
+        }
```

Comment:
Skoro if(...) przyjmuje warunek "true/false" (w tym przypadku "arg >= 18") to oznacza, że "arg >= 18" zwraca true false.
Idąc dalej - można całą tą metodę skrócić do: `return arg >= 18`